deps/snappy/freebsd/snappy-stubs-public.h

```diff
@@ -76,10 +80,21 @@ typedef unsigned long long uint64;
 
 typedef std::string string;
 
+#ifndef DISALLOW_COPY_AND_ASSIGN
 #define DISALLOW_COPY_AND_ASSIGN(TypeName) \
   TypeName(const TypeName&);               \
   void operator=(const TypeName&)
+#endif
+
+#if !0
+// Windows does not have an iovec type, yet the concept is universally useful.
+// It is simple to define it ourselves, so we put it inside our own namespace.
+struct iovec {
+	void* iov_base;
+	size_t iov_len;
+};
+#endif
```

Comment:
It seems to be a bug in [snappy configure.ac#L24](https://github.com/google/snappy/blob/master/configure.ac#L24), without checking `sys/uio.h`, `ac_cv_have_sys_uio_h` will always be **0**.

I'll open a PR there.